src/StagingBanner.jsx

```diff
@@ -19,12 +25,19 @@ const StagingBanner = () => {
           <div
             className={cx('container icon', stagingBannerConfig.extraClasses)}
           >
-            <i aria-hidden="true" className="exclamation circle icon"></i>
+            {stagingBannerConfig.bannerIcon ? (
+              <Icon name={stagingBannerConfig.bannerIcon} size="24px" />
```

Comment:
I would reference the icon import as loadable and load the icon only if it's actually used otherwise by default, we will have an unused import.